ReactCommon/fabric/uimanager/Scheduler.cpp

```diff
@@ -105,9 +105,15 @@ void Scheduler::renderTemplateToSurface(
           [&](const SharedRootShadowNode &oldRootShadowNode) {
             return std::make_shared<RootShadowNode>(
                 *oldRootShadowNode,
-                ShadowNodeFragment{.children =
-                                       std::make_shared<SharedShadowNodeList>(
-                                           SharedShadowNodeList{tree})});
+                ShadowNodeFragment{
+                    0,                                     // tag
```

Comment:
I think you can eliminate the type name here as well, it should be deduced --right?